bitnami/redmine/templates/deployment.yaml

```diff
@@ -126,6 +131,18 @@ spec:
             - name: SMTP_TLS
               value: {{ .Values.smtpTls | quote }}
             {{- end }}
+            {{- if .Values.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsSecret "context" $) }}
+            {{- end }}
           ports:
             - name: http-redmine
               containerPort: 3000
```

Comment:
We should also implement containerPorts.http

Reply:
Done! 👍